app/src/router.js

```diff
@@ -10,6 +10,10 @@ export const routeGuard = (store) => async (to, from, next) => {
   // Set any open modal to false
   store.state.session.currrentModalOpen = false
 
+  // notifications is landing page when user is signed in with email
+  if (to.fullPath.includes("portfolio") && store.state.account.userSignedIn) {
+    next(`/notifications`)
```

Comment:
don't need to return here?

Reply:
no :shrug:

Reply:
vue router docs don't mention a need for returns